app/javascript/packs/verify-flow.tsx

```diff
@@ -75,16 +83,30 @@ const mapKeys = (object: object, mapKey: (key: string) => string) =>
 const storage = new SecretSessionStorage<SecretValues>('verify');
 
 (async () => {
-  const cryptoKey = await crypto.subtle.importKey('raw', storeKey, 'AES-GCM', true, [
-    'encrypt',
-    'decrypt',
-  ]);
-  storage.key = cryptoKey;
-  await storage.load();
-  const userBundleToken = initialValues.userBundleToken as string;
-  await storage.setItem('userBundleToken', userBundleToken);
-  const { pii, metadata } = JSON.parse(atob(userBundleToken.split('.')[1])) as UserBundle;
-  Object.assign(initialValues, Object.fromEntries(mapKeys(pii, camelCase)));
+  let cryptoKey: CryptoKey;
+  let userBundle: UserBundle;
+  try {
+    const storeKey = s2ab(atob(storeKeyBase64));
+    cryptoKey = await crypto.subtle.importKey('raw', storeKey, 'AES-GCM', true, [
+      'encrypt',
+      'decrypt',
+    ]);
+    storage.key = cryptoKey;
+    await storage.load();
+    const userBundleToken = initialValues.userBundleToken as string;
+    await storage.setItem('userBundleToken', userBundleToken);
+    userBundle = JSON.parse(atob(userBundleToken.split('.')[1])) as UserBundle;
+    Object.assign(initialValues, Object.fromEntries(mapKeys(userBundle.pii, camelCase)));
+  } catch (error) {
+    trackError(error);
+    render(
+      <FlowContext.Provider value={{ inPersonURL } as FlowContextValue}>
+        <ErrorStatusPage />
+      </FlowContext.Provider>,
+      appRoot,
+    );
+    return;
+  }
```

Comment:
It would be nice to handle this logic inside the components so that we can have ErrorBoundary dealing with all errors and not have to duplicate as much. Not sure how straight-forward a refactor that would be and if it'd be best to do here vs. a separate pull request.